docs/get_started/index.md

```diff
@@ -1,2 +1,164 @@
+
+### Binary Installer[](#binary-installer)
+
+The process starts by downloading the installer, see below for how to download the most recent release:
+
+1.  Go to [the i8terminal.io website download page](https://www.i8terminal.io/download)
+2.  Click on the `Download For Windows` button in the Download i8 Terminal section
+
+When the file is downloaded, use the following steps to run the I8 Terminal:
+
+**Step 1: Double-click the `.msi` file that got downloaded to your `Downloads` folder**
+
+You will most likely receive the error below stating “Windows protected your PC”. This is because the installer is still in the beta phase, and the team has not yet requested verification from Windows.
+
+[![windows_protected_your_pc](https://www.investoreight.com/media/i8terminal-binaryinstaller-step1.png)](https://www.investoreight.com/media/i8terminal-binaryinstaller-step1.png)
+
+**Step 2: Click on `More info` and select `Run anyway` to start the installation process**
+
+Proceed by following the steps.
+
+[![run_anyway](https://www.investoreight.com/media/i8terminal-binaryinstaller-step2.png)](https://www.investoreight.com/media/i8terminal-binaryinstaller-step2.png)
+
+**Step 3: Select the destination directory you want to install I8 Terminal**
+
+I8 Terminal is installed now!
+
+[![select_destination](https://www.investoreight.com/media/i8terminal-binaryinstaller-step3.png)](https://www.investoreight.com/media/i8terminal-binaryinstaller-step3.png)
+
+### Installing with Python pip[](#installing-with-python-pip)
+
+If you have Python 3 installed, you can simply install the tool with Python pip:
+
+    pip install i8-terminal
+
+We recommend installing the I8 Terminal in an isolated virtual environment. This can be done as follows:
+
+#### On Mac OS or Linux:
+
+    python3 -m venv .venv
+    source .venv/bin/activate
+    pip install i8-terminal
+
+#### On Windows (Using Git Bash):
+
+    python3 -m venv .venv
+    source .venv/Scripts/activate
+    pip install i8-terminal
+
+#### On Windows (Using Command Prompt or PowerShell):
+
+    python3 -m venv .venv
+    .venv\\Scripts\\activate
+    pip install i8-terminal
+
+### Running from Code[](#running-from-code)
+
+The process starts by cloning the code, see below for how to run the terminal from the code:
+
+**Step 1: Clone the repo**
+
+    git clone git@github.com:investoreight/i8-terminal.git
+
+**Step 2: Go to the directory**
+
+    cd i8-terminal
+
+**Step 3: Activate the isolated virtual environment**
+
+#### Using Git Bash:
+
+    python3 -m venv .venv
+    source .venv/Scripts/activate
+
+#### Using Command Prompt or PowerShell:
+
+    python3 -m venv .venv
+    .venv\\Scripts\\activate
+
+**Step 4: Install required libraries**
+
+    pip install -r requirements.dev
```

Comment:
Are you sure this alone would work? You should also include `requirement.txt`. Please test it on a new and clean virtual environment.